utils/utils.c

```diff
@@ -392,3 +392,81 @@ char * strjoin(char *left, char *right, char *delim)
 	strcpy(new + len - rlen - 1, right);
 	return new;
 }
+
+#define QUOTE '\''
+#define DQUOTE '"'
+
+static int setargs(char *args, char **argv)
+{
+	int count = 0;
+
+	/* ignore start spaces */
+	while (isspace(*args))
+		++args;
+
+	while (*args) {
+		/* consider quotes and update argv */
+		if (*args == QUOTE) {
+			if (argv)
+				argv[count] = args + 1;
+			while (*args != QUOTE)
+				++args;
+		} else if (*args == DQUOTE) {
```

Comment:
The coding style saids that the "else" clause should start at the next line of the closing "}".

```
   if () {
      ...
   }
   else if () {
      ...
   }
```